pkg/utils/status.go

```diff
@@ -242,12 +244,14 @@ func calculateState(ctx context.Context, instance *csmv1.ContainerStorageModule,
 	log.Infof("daemonset expected [%d]", expected)
 	log.Infof("daemonset nodeStatus.Available [%s]", nodeStatus.Available)
 
-	if (controllerReplicas == controllerStatus.Available) && (fmt.Sprintf("%d", expected) == nodeStatus.Available) {
+	modrunning, err := statusForAppMob(ctx, instance, r, newStatus)
```

Comment:
Will this impact drivers and other modules?
This method returns a boolean running = AppRunning && VelRunning && CertRunning && RestRunning
that won't be calculated if AM is not the module being installed

Reply:
Thats a good catch 
i had missed a logic while merging branch
I have added it back and i know this will work 
let me know if this works for you